### PR TITLE
tidb-dashboard: add doc for creating SQL binding from statement history 

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-statement-details.md [hunk 1]
```diff
@@ -8,12 +8,50 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
-- SQL 语句概况：包括 SQL 模板、SQL 模板 ID、当前查看的时间范围、执行计划个数以及执行所在的数据库（下图区域 1）
+- SQL 语句概况：包括 SQL 模板、SQL 模板 ID、当前查看的时间范围、执行计划个数、执行所在的数据库以及快速绑定执行计划功能（下图区域 1）
 - 执行计划列表：如果一个 SQL 语句有多个执行计划，则显示执行计划列表。除了文本信息显示，TiDB 自 v6.2.0 开始引入图形化执行计划，通过图形化的执行计划，你可以更清晰地了解一个语句的具体算子和对应的内容。可以选择不同的执行计划，在列表和图形下方会显示该执行计划详情（下图区域 2）
 - 执行计划详情：显示选中的执行计划的详细信息，具体见下一小节（下图区域 3）
 
 ![详情](/media/dashboard/dashboard-statement-detail-v620.png)
 
+## 快速绑定执行计划
+
+在 v6.6 及以上版本中，引入了快速绑定执行计划功能：用户可以在 TiDB Dashboard 中，一分钟内完成 SQL 与特定计划的绑定。
```
Comment (ran-huang) [inline dashboard/dashboard-statement-details.md:19]:
```suggestion
在 v6.6.0 及以上版本中，引入了快速绑定执行计划功能。用户可以在 TiDB Dashboard 中完成 SQL 语句与特定计划的绑定，耗时约 1 分钟。
```

Comment (lilyjazz) [reply dashboard/dashboard-statement-details.md:19]: 实测很快，一秒都不要..我直接把 一分钟 描述删除掉。

File: dashboard/dashboard-statement-details.md [hunk 2]
```diff
@@ -8,12 +8,50 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
 
 在列表中点击任意一行可以进入该 SQL 语句的详情页查看更详细的信息，此信息包括三大部分：
 
-- SQL 语句概况：包括 SQL 模板、SQL 模板 ID、当前查看的时间范围、执行计划个数以及执行所在的数据库（下图区域 1）
+- SQL 语句概况：包括 SQL 模板、SQL 模板 ID、当前查看的时间范围、执行计划个数、执行所在的数据库以及快速绑定执行计划功能（下图区域 1）
```
[the remark this answers is not in the document]
Comment (lilyjazz) [reply dashboard/dashboard-statement-details.md:11]: 图片补上了

File: dashboard/dashboard-statement-details.md [hunk 3]
```diff
@@ -8,12 +8,50 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
 - 执行计划列表：如果一个 SQL 语句有多个执行计划，则显示执行计划列表。除了文本信息显示，TiDB 自 v6.2.0 开始引入图形化执行计划，通过图形化的执行计划，你可以更清晰地了解一个语句的具体算子和对应的内容。可以选择不同的执行计划，在列表和图形下方会显示该执行计划详情（下图区域 2）
 - 执行计划详情：显示选中的执行计划的详细信息，具体见下一小节（下图区域 3）
 
 ![详情](/media/dashboard/dashboard-statement-detail-v620.png)
 
+## 快速绑定执行计划
+
+在 v6.6 及以上版本中，引入了快速绑定执行计划功能：用户可以在 TiDB Dashboard 中，快速完成 SQL 与特定计划的绑定。
```
Comment (ran-huang) [inline dashboard/dashboard-statement-details.md:19]:
```suggestion
自 v6.6.0 起，TiDB 引入了快速绑定执行计划的功能。你可以在 TiDB Dashboard 中，快速完成 SQL 语句与特定计划的绑定。
```

File: dashboard/dashboard-statement-details.md [hunk 4]
```diff
@@ -8,12 +8,50 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+### 使用方式
+
+#### 绑定一个执行计划
+
+1. 点击**执行计划绑定**，弹出弹窗。
```
Comment (ran-huang) [inline dashboard/dashboard-statement-details.md:25]: 这里图片上是 **Plan Binding**，只写中文的话有点不太好找，改成这样可以吗，兼顾中英文

```suggestion
1. 点击**执行计划绑定** (Plan Binding)，弹出弹窗。
```

File: dashboard/dashboard-statement-details.md [hunk 5]
```diff
@@ -8,12 +8,50 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+![快速绑定执行计划-未绑定-入口](/media/dashboard/dashboard-quick-binding-entry-notbound.png)
+
+2. 选择一个需要绑定的执行计划，点击**绑定**。
```
Comment (ran-huang) [inline dashboard/dashboard-statement-details.md:29]:
```suggestion
2. 选择一个需要绑定的执行计划，点击**绑定** (Binding)。
```

File: dashboard/dashboard-statement-details.md [hunk 6]
```diff
@@ -8,12 +8,50 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+![快速绑定执行计划-弹窗](/media/dashboard/dashboard-quick-binding-popup-notbound.png)
+
+3. 绑定成功，可以看到**已绑定**提示。
```
Comment (ran-huang) [inline dashboard/dashboard-statement-details.md:33]:
```suggestion
3. 绑定成功，可以看到**已绑定** (Bound) 提示。
```

File: dashboard/dashboard-statement-details.md [hunk 7]
```diff
@@ -8,12 +8,50 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+![快速绑定执行计划-弹窗-绑定成功](/media/dashboard/dashboard-quick-binding-popup-bound.png)
+
+#### 取消已有的执行计划绑定
+
+1. 当 SQL 已经绑定了执行计划后，点击**执行计划绑定**，弹出弹窗。
```
Comment (ran-huang) [inline dashboard/dashboard-statement-details.md:39]:
```suggestion
1. 当 SQL 已经绑定了执行计划后，点击**执行计划绑定** (Plan Binding)，弹出弹窗。
```

File: dashboard/dashboard-statement-details.md [hunk 8]
```diff
@@ -8,12 +8,50 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+![快速绑定执行计划-已绑定-入口](/media/dashboard/dashboard-quick-binding-entry-bound.png)
+
+2. 点击**取消绑定**。
```
Comment (ran-huang) [inline dashboard/dashboard-statement-details.md:43]:
```suggestion
2. 点击**取消绑定** (Drop)。
```

File: dashboard/dashboard-statement-details.md [hunk 9]
```diff
@@ -8,12 +8,50 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+![快速绑定执行计划-弹窗-绑定成功](/media/dashboard/dashboard-quick-binding-popup-bound.png)
+
+3. 取消成功，可以看到**未绑定**提示。
```
Comment (ran-huang) [inline dashboard/dashboard-statement-details.md:47]:
```suggestion
3. 取消成功，可以看到**未绑定** (Not Bound) 提示。
```

File: dashboard/dashboard-statement-details.md (hunk 4 above)
Comment (YiniXu9506) [reply dashboard/dashboard-statement-details.md:25]: 这个页面有中英文两种语言，如果中英文文档的截图内容不严格区分语言的话，这里**执行计划绑定** 可以带上 Plan Binding 标注

File: dashboard/dashboard-statement-details.md (hunk 5 above)
Comment (YiniXu9506) [reply dashboard/dashboard-statement-details.md:29]: 此处「绑定」按钮是一个动作，如有英文标注，应该用 Bind。

File: dashboard/dashboard-statement-details.md (hunk 4 above)
Comment (ran-huang) [reply dashboard/dashboard-statement-details.md:25]: 图片可以保持不变~ 加上文字就好了

**Review by YiniXu9506 — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

File: dashboard/dashboard-statement-details.md [hunk 10]
```diff
@@ -8,11 +8,54 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+自 v6.6.0 起，TiDB 引入了快速绑定执行计划的功能。你可以在 TiDB Dashboard 中，快速完成 SQL 语句与特定计划的绑定。
+
+### 使用方式
+
+#### 绑定一个执行计划
+
+1. 点击**执行计划绑定** (Plan Binding)，弹出弹窗。
+
+![快速绑定执行计划-未绑定-入口](/media/dashboard/dashboard-quick-binding-entry-notbound.png)
```
Comment (TomShawn) [inline dashboard/dashboard-statement-details.md:27]:
```suggestion
    ![快速绑定执行计划-未绑定-入口](/media/dashboard/dashboard-quick-binding-entry-notbound.png)
```

File: dashboard/dashboard-statement-details.md [hunk 11]
```diff
@@ -8,11 +8,54 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+2. 选择一个需要绑定的执行计划，点击**绑定** (Bind)。
+
+![快速绑定执行计划-弹窗](/media/dashboard/dashboard-quick-binding-popup-notbound.png)
```
Comment (TomShawn) [inline dashboard/dashboard-statement-details.md:31]:
```suggestion
    ![快速绑定执行计划-弹窗](/media/dashboard/dashboard-quick-binding-popup-notbound.png)
```

File: dashboard/dashboard-statement-details.md [hunk 12]
```diff
@@ -8,11 +8,54 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+3. 绑定成功后，可以看到**已绑定** (Bound) 提示。
+
+![快速绑定执行计划-弹窗-绑定成功](/media/dashboard/dashboard-quick-binding-popup-bound.png)
```
Comment (TomShawn) [inline dashboard/dashboard-statement-details.md:35]:
```suggestion
    ![快速绑定执行计划-弹窗-绑定成功](/media/dashboard/dashboard-quick-binding-popup-bound.png)
```

File: dashboard/dashboard-statement-details.md [hunk 13]
```diff
@@ -8,11 +8,54 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+#### 取消已有的执行计划绑定
+
+1. 在已经绑定了执行计划的 SQL 语句页面上，点击**执行计划绑定** (Plan Binding)，弹出弹窗。
+
+![快速绑定执行计划-已绑定-入口](/media/dashboard/dashboard-quick-binding-entry-bound.png)
```
Comment (TomShawn) [inline dashboard/dashboard-statement-details.md:41]:
```suggestion
    ![快速绑定执行计划-已绑定-入口](/media/dashboard/dashboard-quick-binding-entry-bound.png)
```

File: dashboard/dashboard-statement-details.md [hunk 14]
```diff
@@ -8,11 +8,54 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+2. 点击**取消绑定** (Drop)。
+
+![快速绑定执行计划-弹窗-绑定成功](/media/dashboard/dashboard-quick-binding-popup-bound.png)
```
Comment (TomShawn) [inline dashboard/dashboard-statement-details.md:45]:
```suggestion
    ![快速绑定执行计划-弹窗-绑定成功](/media/dashboard/dashboard-quick-binding-popup-bound.png)
```

File: dashboard/dashboard-statement-details.md [hunk 15]
```diff
@@ -8,11 +8,54 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-statement-details/','/docs-cn/dev/da
...
+
+3. 取消成功后，可以看到**未绑定** (Not Bound) 提示。
+
+![快速绑定执行计划-弹窗](/media/dashboard/dashboard-quick-binding-popup-notbound.png)
```
Comment (TomShawn) [inline dashboard/dashboard-statement-details.md:49]:
```suggestion
    ![快速绑定执行计划-弹窗](/media/dashboard/dashboard-quick-binding-popup-notbound.png)
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**